### ISSUE TITLE
Readme

### ISSUE THREAD

**Issue by jkoper** [Entroper/Randomizers#19]
This line from the readme should be updated:  "Speaking of classes, the TAIL can be anywhere -- including the Temple of Fiends Revisited."  Also, you have a line about permissions being a potential future change, which should probably be removed and replaced with a section on permissions.

Side question:  when early canoe becomes possible, will this allow the ruby to be in any of the places accessible by canoe?

**Comment by Entroper** [comment]
The README will get a rewrite as soon as I get back from vacation.  :)

About the canoe, I think it's possible to do that.  I have to think hard about it and make sure I don't break the game some other way, because you could get the airship as soon as you have TNT+FLOATER, which would mean the RUBY could be almost anywhere.  Actually, you could just land at Sarda's Cave at that point.

**Comment by jkoper** [comment]
It doesn't seem too hard to write the dependencies.  I have complete faith that you could, but I'm willing to look over any logic you have.  Regarding the canoe itself, do you have a plan for changing the check?  XioMeiLin looked into it and said the old man is looking at 6035 in the RAM for an 01 before he'll give you the canoe.  Not sure if it's possible to just make him look somewhere that's always a 1 instead.

**Comment by Jaylow7** [comment]
I'd think it should be possible to change what he looks for. Instead of the trigger being Lich dead, make it something that you 100% will have done by that point, like Garland being dead or having the Ship.

**Comment by Entroper** [comment]
The check itself is pretty easy to overwrite with NOPs.  Same thing I did for Sarda.  The CPU will just skip over it.

**Comment by Entroper** [comment]
It's interesting, with Early Canoe, you could fly to Sarda's Cave and get what's in those chests, which would make them unblocked by the RUBY.  The way I've modeled treasure blocking doesn't allow for chests to be unblocked by more than one quest item, so it may require a rewrite to do Early Canoe properly.  I think I know how to do it, and it might even be easier to understand than the current model.

**Comment by Entroper** [comment]
I've updated the README to include all planned features for 1.0.  I also opened a separate issue for Early Canoe.
